src/praisonai-deploy/praisonai_deploy/providers/azure.py

```
@@ -145,10 +144,24 @@ def deploy(self) -> DeployResult:
+                    '--name', self.config.service_name,
+                    '--resource-group', self.config.resource_group,
+                    '--image', self.config.image or f"{self.config.service_name}:latest",
+                    '--cpu', str(self.config.cpu),
+                    '--memory', f"{self.config.memory}Gi",
+                    '--min-replicas', str(self.config.min_instances),
+                    '--max-replicas', str(self.config.max_instances),
+                ]
+                if self.config.env_vars:
+                    update_cmd.append('--set-env-vars')
+                    update_cmd.extend(f"{k}={v}" for k, v in self.config.env_vars.items())
                 result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)